settings/preview-deployments.mdx

```diff
@@ -5,20 +5,51 @@ icon: "scan-eye"
+Preview deployments let you see how changes to your docs will look before merging to production. Each preview creates a shareable URL that updates automatically as you push new changes.
 
-If you have created a pull or merge request when using Git, the generated preview URL will be available as a comment from the Mintlify bot in the PR. This URL will always reflect the latest deployment changes.
+## Creating preview deployments
 
-## Sharing Preview Deployments
+Preview deployments are created automatically through pull requests or manually from your dashboard.
 
-After generating a preview URL, you can share that link with anybody for them to view.
+### Automatic previews
 
-### Authenticated Preview Deployments
+When you create a pull request, the Mintlify bot automatically adds a link to view the preview deployment in your pull request. The preview updates each time you push new commits to the branch.
 
-If you'd like to restrict access to your Preview Deployments, you can set up preview deployment authentication. When using this authentication method, only Mintlify authenticated users will be able to access your Preview Deployments.
+<Frame>
+  <img src="/images/previews/preview-deployment-light.png" alt="Link to view deployment in the pull request timeline" className="block dark:hidden" />
+  <img src="/images/previews/preview-deployment-dark.png" alt="Link to view deployment in the pull request timeline" className="hidden dark:block" />
+</Frame>
 
-To set up preview deployment authentication, please [contact sales](mailto:sales@mintlify.com).
+### Manual previews
+
+You can manually create a preview for any branch.
+
+1. Go to your [dashboard](https://dashboard.mintlify.com/).
+2. Select **Previews**.
+3. Select **Create preview**.
+4. Enter your branch name.
+5. Select **Create deployment**.
+
+## Redeploying a preview
+
+Redeploy a preview to refresh content or retry after a failed deployment.
+
+1. Select the preview from your [dashboard](https://dashboard.mintlify.com/).
+2. Select **Redeploy**.
+
+<Frame>
+  <img src="/images/previews/redeploy-preview-light.png" alt="The Previews menu with the deploy button emphasized by an orange rectangle." className="block dark:hidden" />
+  <img src="/images/previews/redeploy-preview-dark.png" alt="The Previews menu with the deploy button emphasized by an orange rectangle." className="hidden dark:block" />
+</Frame>
+
+## Sharing preview deployments
+
+Preview deployments can be publicly accessible or restricted to authenticated users.
+
+**Public access**: Preview URLs are publicly viewable by default. Share a preview link with anyone who needs to review your changes.
+
+**Restricted access**: Enable deployment authentication to limit preview access to authenticated Mintlify users only. [Contact sales](mailto:sales@mintlify.com) to set up authentication for your previews.
```

Comment:
we should make the distinction that preview deployment auth is different from normal auth (authenticating ur preview deployments vs. ur main docs)
```suggestion
**Restricted access**: Enable preview deployment authentication to limit preview access to authenticated Mintlify users only. [Contact sales](mailto:sales@mintlify.com) to set up authentication for your previews.
```